components/KeyFactIndicator/KeyFactIndicator.tsx

```diff
@@ -32,36 +32,37 @@ const KeyFactIndicator: FunctionComponent<KeyFactIndicatorProps> = ({ indicator,
         <div className="l-2up-3up__col">
           {contentOptions.map(({ stat, chart }, index) => {
             if (stat) {
+              const suffix = stat.valueSuffix || indicator.value_suffix;
+
               return (
                 <IndicatorStat
                   key={index}
                   heading={processTemplateString(stat.title || '', templateOptions)}
                   meta={stat.meta || { description: indicator.description, source: indicator.source }}
                 >
-                  <ErrorBoundary>
```

Comment:
You removed the ErrorBoundary, probably during a rebase. Please add it back.

Reply:
Yea rebasing removed this